src/nupic/algorithms/Cells4.hpp

```diff
@@ -406,8 +407,8 @@ namespace nupic {
                Real permInc =.1,
                Real globalDecay =0,
                bool doPooling =false,
-               int seed =-1,
-               bool doItAll =false,
+               int seed =42,
```

Comment:
Please revert change to default seed. This changes the behavior:
https://github.com/numenta/nupic.core/blob/1f668c644e4f9429b04fb7a27b236dcd4ee99a09/src/nupic/algorithms/Cells4.cpp#L84